### DISCUSSION TITLE
Back to the top button

### DISCUSSION THREAD

**Discussion by Dev00kushal** [JulianPrieber/littlelink-custom#616] [Feature requests]
It would be great if we could add a back to the top button at the end of our landing page. As the user needs to scroll up which is looking so much hectic for the website.

A good example would be something like this:
![image](https://github.com/LinkStackOrg/LinkStack/assets/127578046/792e3367-943a-4ef3-8b30-9b7b89f66169)

for our website which is currently like this :
![image](https://github.com/LinkStackOrg/LinkStack/assets/127578046/75b05f2f-fd83-4409-bf92-660ca33d65b8)

**Comment by JulianPrieber** [comment]
![chrome_OSRZUA9qPA](https://github.com/LinkStackOrg/LinkStack/assets/60265788/66ddcd69-4e90-48f1-92f8-c38a2dd928fd)

We do have a back to top button at the end of our landing page.

**Reply by Dev00kushal** [reply]
Okay, it makes sense, but isn't it less visually striking than a button that would show the user more vision? How do you feel?
